modules/ROOT/pages/connected-apps-overview.adoc

```diff
@@ -0,0 +1,36 @@
+= Connected Apps
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can connect apps to Anypoint Platform without granting them unrestricted access to data or having to share sensitive credentials.
+When you grant an app access, you have direct control over its data access, usage, and credentials.
+You can audit the use of data by connected apps, and you can also revoke access at any time.
+
+== Authentication
+
+Use OAuth 2.0 and OpenID to authorize apps to access resources using tokens or Single Sign-On (SSO).
+Additionally, you can share account data with third-party applications.
+For more information on which users can grant access to applications using Anypoint Platform, see Connected Apps Permissions by User.
+
+== First-party and Third-party Applications
+
+To determine whether the app you want to connect is a first- or third-party application, consider the below:
```

Comment:
This reads weird. There's multiple personas involved here -- There's an org admin who creates first-party clients, and whitelists third-party clients, and there's a user who authorizes a client application (i.e. connects the application). Users do not need to know or care if the app is first- or third-party.